src/components/manager/ManagerAnalysisContent.js

```diff
@@ -2,15 +2,30 @@ import React from "react";
 import { useDispatch } from "react-redux";
 import { useNavigate } from "react-router-dom";
 
+import { analyzeData, RESOURCE_EVERYTHING } from "bento-auth-js";
+
+import { useResourcePermissionsWrapper } from "@/hooks";
 import { submitAnalysisWorkflowRun } from "@/modules/wes/actions";
 
+import ForbiddenContent from "./ForbiddenContent";
 import RunSetupWizard from "./RunSetupWizard";
 import RunSetupConfirmDisplay from "./RunSetupConfirmDisplay";
 
 const ManagerAnalysisContent = () => {
     const dispatch = useDispatch();
     const navigate = useNavigate();
 
+    const { permissions, hasAttemptedPermissions } = useResourcePermissionsWrapper(RESOURCE_EVERYTHING);
+
+    // TODO: each workflow should have definitions for permissions scopes, so we can instead check if we can run at
+    //  least one workflow.
+
+    if (hasAttemptedPermissions && !permissions.includes(analyzeData)) {
+        return (
+            <ForbiddenContent message="You do not have permission to view the analysis wizard." />
+        );
+    }
```

Comment:
Can be simplified with `useHasResourcePermissionWrapper(RESOURCE_EVERYTHING, analyzeData)`.

Reply:
doesn't include a `hasAttempted` attr, so it doesn't work for this case.

Reply:
(should do in bento auth js to support this)

Reply:
Ah true, if we add `hasAttempted` we could even wrap this in an authz component RequirePermissions, similar to RequireAuth and AutoAuthenticate.